src/Humanizer/Properties/Resources.resx

```diff
@@ -711,4 +711,28 @@
   <data name="DataUnit_TerabyteSymbol" xml:space="preserve">
     <value>TB</value>
   </data>
+  <data name="TimeUnit_Millisecond" xml:space="preserve">
+    <value>ms</value>
+  </data>
+  <data name="TimeUnit_Second" xml:space="preserve">
+    <value>s</value>
+  </data>
+  <data name="TimeUnit_Minute" xml:space="preserve">
+    <value>min</value>
+  </data>
+  <data name="TimeUnit_Hour" xml:space="preserve">
+    <value>h</value>
+  </data>
+  <data name="TimeUnit_Day" xml:space="preserve">
+    <value>d</value>
+  </data>
+  <data name="TimeUnit_Week" xml:space="preserve">
+    <value>week</value>
+  </data>
+  <data name="TimeUnit_Month" xml:space="preserve">
+    <value>mo</value>
+  </data>
+  <data name="TimeUnit_Year" xml:space="preserve">
+    <value>a</value>
```

Comment:
According to Wikipedia, `a` is a common year unit symbol in English, but `y` is apparently common, too. Maybe a native speaker could weigh in on this?